skimage/transform/tests/test_orientation.py

```diff
@@ -0,0 +1,139 @@
+import numpy as np
+from numpy import pi, sin, cos
+from numpy.testing import (assert_equal, assert_almost_equal,
+                           assert_array_almost_equal)
+
+from skimage.transform._orientation import (convert_quasipolar_coords,
+                                            compute_rotation_matrix,
+                                            _normalize, _axis_0_rotation_matrix)
+
```

Comment:
Could you please document the tests? What we are testing and the expected output would be great.